File: java/kode program/Electronic.java
public class Electronic extends Product {
    private String brand;
    private String warranty;
    private String category;

    public Electronic(int id, String name, int price, int stock, String image,
                      String brand, String warranty, String category) {
        super(id, name, price, stock, image);
        this.brand = brand;
        this.warranty = warranty;
        this.category = category;
    }

    // setter dan getter untuk brand
    public void setBrand(String brand) {
        this.brand = brand;
    }
    public String getBrand() {
        return this.brand;
    }

    // setter dan getter untuk warranty
    public void setWarranty(String warranty) {
        this.warranty = warranty;
    }
    public String getWarranty() {
        return this.warranty;
    }

    // setter dan getter untuk category
    public void setCategory(String category) {
        this.category = category;
    }
    public String getCategory() {
        return this.category;
    }

}

File: python/kode program/Electronic.py
from Product import Product

# Electronic mewarisi dari Product
class Electronic(Product):
    def __init__(self, id, name, price, stock, image, brand, warranty, category):
        # panggil constructor dari Product
        super().__init__(id, name, price, stock, image)
        self._brand = brand
        self._warranty = warranty
        self._category = category

    # setter dan getter untuk brand
    def set_brand(self, brand):
        self._brand = brand

    def get_brand(self):
        return self._brand

    # setter dan getter untuk warranty
    def set_warranty(self, warranty):
        self._warranty = warranty

    def get_warranty(self):
        return self._warranty

    # setter dan getter untuk category
    def set_category(self, category):
        self._category = category

    def get_category(self):
        return self._category
